spec/shared/shared_examples/user_not_login.rb

```diff
@@ -0,0 +1,8 @@
+shared_examples "user_not_login" do
```

Comment:
đưa cái before vô đây  luôn đi xong ngoài kia không cần đưa vô context not login nửa mà chỉ it_behave_like .... thôi

Reply:
@dongnd-3156 dạ okie anh để em update lại ạ

Reply:
```ruby
context "when not_login" do
 let(:expect_message) { "You are not authorized to access this page." }

  before { subject }

  it do
    expect(response).to redirect_to root_path
    expect(flash[:error]).to eq expect_message
  end
end
```

Reply:
Dạ okie anh